assets/js/googlesitekit/widgets/components/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Widget components.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Widget from './Widget';
+
+export { default as WidgetAreaRenderer } from './WidgetAreaRenderer';
+export { default as WidgetContextRenderer } from './WidgetContextRenderer';
+export { default as WidgetRenderer } from './WidgetRenderer';
+
+export { Widget };
+
+export default {
+	components: { Widget },
+};
```

Comment:
Why do we need to export this object here? I think we should just export all components (like you're doing above) like elsewhere. Which components specifically get exposed on the public `googlesitekit.widgets` object should be handled in `assets/js/googlesitekit/widgets/index.js` I think, with the rest of that object. (I believe as far as I can tell this part is still missing from here at this point.)